config.go

```diff
@@ -124,6 +124,11 @@ type Config struct {
 	// AdditionalGroups specifies the gids that should be added to supplementary groups
 	// in addition to those that the user belongs to.
 	AdditionalGroups []int `json:"additional_groups,omitempty"`
+	// UidMappings is an array of User ID mappings for User Namespaces
+	UidMappings []IDMap `json:"uid_mappings,omitempty"`
```

Comment:
According to google style guide all this things should be like `UIDMappings` and `GIDMappings`, but I'm not insist :)